src/lib/Libutil/execvnode_seq_util.c

```diff
@@ -554,6 +574,9 @@ get_execvnodes_count(char *str)
 	if (str == NULL)
 		return 0;
 
+	if (str[0] == '(')
```

Comment:
can you please add a comment here to explain why you are checking for parenthesis?

Reply:
I updated the doxygen comment for the expected input.